src/preload.ts

```diff
@@ -83,12 +83,24 @@ window.loadTables = (signal: AbortSignal) =>
     return result;
   });
 
+window.getAnonymizedAccessDbId = (signal: AbortSignal) =>
+  newTask(signal, async (taskId) => {
+    const result = await ipcRenderer.invoke('get_anonymized_access_db_id', taskId);
+    return result;
+  });
+
 window.removeTable = (tableName: string, signal: AbortSignal) =>
   newTask(signal, async (taskId) => {
     const result = await ipcRenderer.invoke('remove_table', taskId, tableName);
     return result;
   });
 
+window.buildSampleCardEncoded = (tableName: string, aidColumns: string[], signal: AbortSignal) =>
```

Comment:
I think we should start grouping window IPCs into objects:

```
window.metabase.getAnonymizedAccessDbId(...)
window.metabase.buildSampleCardEncoded(...)
```

Reply:
I'll leave myself a self-note to handle this in a follow-up, to not inflate the PR with refactors

Reply:
Forgot that we show the window before services are ready so the above is impossible :confused:

Reply:
we're getting rid of it completely, we don't need it anymore at this point, see other thread.